public/full.js

```diff
@@ -831,6 +831,10 @@ window.unmuteVideoMember = () => {
   roomObj.videoUnmute({ memberId: id })
 }
 
+window.removeMember = (id) => {
+  roomObj.removeMember({ memberid: id})
```

Comment:
If the idea is to test a feature that is not exposed in the SDK already, you could do

`const result = await window.__client.__wsClient.wsClient.execute(params)` 

Where `params` is the JSON-RPC request.